test/arithmetic/values/fixed_point_value_test.dart

```diff
@@ -291,4 +291,31 @@ void main() {
       }
     }
   });
+
+  test('FixedPointValue: toFloatingPointValue signed', () {
+    const m = 10;
+    const n = 12;
+    const width = m + n + 1; // 1 for sign bit
+    for (var i = 0; i < pow(2, width); i++) {
+      final fxv = FixedPointValue.populator(
+              integerWidth: m, fractionWidth: n, signed: true)
+          .ofLogicValue(LogicValue.ofInt(i, width));
+      final fpv = fxv.toFloat();
+      expect(fpv.toDouble(), fxv.toDouble(),
+          reason: 'toFloatingPointValue failed for $i');
+    }
+  });
+
+  test('FixedPointValue: toFloatingPointValue unsigned', () {
```

Comment:
You could add the signage as a loop in previous test and have one test.

Testing for m = 0, and n = 0 as two important corners.
```dart
for (final (m,n) in [(10,12), (0, 22), (22,0)] {
```

Reply:
I wasn't sure what feedback you wanted:  m=0, n=0 should probably throw.